react-function-components/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function CustomButton(props) {
+  return <button>Click Me</button>;
+}
+
+ReactDOM.render(CustomButton(), document.querySelector('#root'));
```

Comment:
Don't call your components when you render them, React will call them for you each time it needs to create a new render: https://reactjs.org/docs/components-and-props.html#rendering-a-component

Reply:
changed to <CustomButton/> from CustomButton()